### ISSUE TITLE
i18n not working when set enableGridMenu = false and enableColumnPicker=false

### ISSUE THREAD

**Issue by JackoPham** [ghiscoding/Angular-Slickgrid#268]
Hi guys. When I use i18n in the Slickgrid , it's not working when I set **enableGridMenu = false and enableColumnPicker** after that, I have got some bug. you can see below.

 const originalCustomItems = Array.isArray(this._userOriginalGridMenu.customItems) && this._userOriginalGridMenu.customItems || [];

I think when I disable enableGridMenu  or enableColumnPicker so that thing will undefined (this._userOriginalGridMenu) 

![image](https://user-images.githubusercontent.com/33243797/63213759-664b8500-c13a-11e9-8122-a69f02d2db31.png)

 "angular-slickgrid": "^2.9.9",
 "@ngx-translate/core": "^11.0.1",
  "@ngx-translate/http-loader": "^4.0.0",

Note: It's good work when **set enableGridMenu = true and enableColumnPicker=true**

Please help me.
Thanks.

**Comment by ghiscoding** [comment]
I added some type checks in the code to avoid trying to access the object when it's null, but technically speaking it's not possible to have this error (at least not with current code). When `enableGridMenu` is set to False, it won't register the `GridMenuExtension`, so it's not supposed to go at all on that line of code, so again I'm not sure how that happens but I added some type checks anyway. There was a lot of refactoring that I've done in the past couple weeks when I worked on the upcoming features, so perhaps this was already fixed in current code, but it's always good to have full type checking anyway, so I will be pushing a new version tonight with a few new features and fixes.
